Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1010,7 +1008,7 @@ public static void HandleCI(Session session, params string[] parameters)
             WorldObject testContainer = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
             if (testContainer.ItemCapacity > 1)
             {
-                var loot = LootGenerationFactory.CreateTestContainerObject(session.Player, weenieId);
+                Container loot = LootGenerationFactory.CreateTestContainerObject(session.Player, weenieId);
                 loot.ContainerId = session.Player.Guid.Full;
                 session.Player.AddToInventory(loot);
```

Comment:
I did not go back and retrofit old code.   That has been in there forever.   I can put it in an actionQueue.